azkaban-common/src/test/java/azkaban/executor/MockExecutorLoader.java

```diff
@@ -122,6 +122,9 @@ public void uploadExecutableNode(ExecutableNode node, Props inputParams)
   @Override
   public void updateExecutableNode(ExecutableNode node)
       throws ExecutorManagerException {
+    if (!nodes.containsKey(node.getId())) {
+      uploadExecutableNode(node, new Props());
+    }
```

Comment:
Yes, but I hope that wouldn't block getting these tests into master. This only started happening after some recent changes merged into master.